scripts/process-icons.js

```diff
@@ -39,9 +39,17 @@ const processIcon = (srcPath, fd, scaleWidth, scaleHeight) => {
 };
 
 // load our license file
-const license = fs.readFileSync(
-    path.join(__dirname, '..', 'config', 'license.js')
-);
+let license;
```

Comment:
TODO: This task seems to be done in a few places so might be good to pull these out into a utilities file to streamline fetching the data (that way if paths change, etc. we can manage that update in one place).

Reply:
I don't count this as a blocker, but this is a good call.